test_validator.js

```
@@ -0,0 +1,87 @@
+        return username && typeof username === 'string' && username.trim().length > 0;
+    }
+
+    static isValidUrl(url) {
+        try {
+            new URL(url);
+            return true;
+        } catch {
+            return false;
+        }
+    }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)